app/models/group.rb

```diff
@@ -17,6 +17,10 @@ def to_param
     external_id
   end
 
+  def creator
+    User.find_by(id: creator_id)
+  end
+
```

Comment:
Why not make this a proper association?

Reply:
I think I was distracted by wondering if we should be doing this via the `Membership` model...there is no good reason not to, so I've changed it to a `belongs_to` association.